jenkins/pipelines/Jenkinsfile

```diff
@@ -1,35 +1,29 @@
 // Manage Jenkins master service
+@Library('Utils') _
+
 pipeline {
     agent any
-    /*
+
     triggers {
         pollSCM('H/15 * * * *')
     }
-    */
+
     stages {
-        stage('Checkout') {
-            steps {
-                // ignore checkout part of excludedRegions
-                checkout([$class: 'GitSCM',
-                    branches: [[name: "*/master"]],
-                    doGenerateSubmoduleConfigurations: false,
-                    extensions: [[
-                        $class: 'PathRestriction',
-                        includedRegions: '^jenkins/.*']],
-                    submoduleCfg: [],
-                    userRemoteConfigs: [[
-                        url: 'https://github.com/RHsyseng/jenkins-on-openshift']]])
-            }
-        }
         stage('S2I build Jenkins with custom configuration') {
+            when {
+                expression {
+                    findFileChangesContains("jenkins")
+                }
+
+            }
             steps {
                 openshiftBuild bldCfg: 'jenkins-custom'
             }
         }
     }
     post {
         failure {
-            mail to: 'aweiteka@redhat.com',
+            mail to: 'example@example.com',
```

Comment:
There are variables in the ansible playbook that set these (`notify_email_list`) and we should use these... but that would be part of a different PR.